src/core/Animators/qrealanimator.cpp

```diff
@@ -55,15 +55,15 @@ void QrealAnimator::prp_setupTreeViewMenu(PropertyMenu * const menu) {
 
     const PropertyMenu::PlainSelectedOp<QrealAnimator> sOp =
     [](QrealAnimator * aTarget) {
-        const auto& intrface = DialogsInterface::instance();
-        intrface.showExpressionDialog(aTarget);
+        const auto& interface = DialogsInterface::instance();
```

Comment:
Can't use `interface`, it's reserved in MSVC (Windows).

https://github.com/friction2d/friction/actions/runs/12444585488/job/34745004655

Reply:
Can you suggest an alternative that won't trigger a false positive typo ?

Reply:
I usually use `iface`.